azavea/echo-locator
# Feature/kak/data update#256

## Overview

Assorted small tasks to finish up:
 - data update, same format as previous
 - add Google Analytics page tracking with `react-ga`
 - add back login page header text, now we have the text
 - shuffle around the detail view links


### Demo

Sign-in page:
![image](https://user-images.githubusercontent.com/960264/60356543-42bc5780-999f-11e9-9ecd-cbe45639e4df.png)

Debug output from Google Analytics, when initialized with `{debug: true}`:
![echo_google_analytics_debug](https://user-images.githubusercontent.com/960264/60356488-1e607b00-999f-11e9-9a1f-e19aa8adb27a.png)

Detail page links section ~~(the spacing looks off)~~:

![image](https://user-images.githubusercontent.com/960264/60356659-90d15b00-999f-11e9-801e-ea9b75e2f0d6.png)


## Notes

~~After shuffling around the detail page links, the margins look off.~~


## Testing Instructions

 * Set the `GOOGLE_ANALYTICS_ID` in `taui/configurations/default/env.yml`
 * `cd taui && yarn install && yarn start`
 * Network requests should go to Google Analytics on page change between map, edit profile, and search
 * Login page should have message in header text
 * Details should no longer have Section 8 link and static link to Apartments.com should work


Closes #256 
Closes #253 
Closes #254 
Closes #255 
Closes #248
